test/JDBC/src/main/java/com/sqlsamples/Config.java

```diff
@@ -68,26 +64,15 @@ static Properties readConfig() {
         return prop;
     }
 
-    //configure properties of logger
-    public static void configureLogger(Logger logger, String logFileName) throws IOException {
-
-        if(compareWithFile){
-            //disable logging if we are comparing against expected output
-            logger.setLevel(Level.OFF);
-        } else {
-            logger.setLevel(Level.DEBUG);
-            PatternLayout pattern = new PatternLayout("%m%n");
-            FileAppender fileAppender = new FileAppender(pattern, logFileName+ ".log");
-            logger.addAppender(fileAppender);
-
-            if(printLogsToConsole){
```

Comment:
why are we removing the printLogsToConsole block here?

Reply:
Earlier the framework had two modes of execution but now there is only one (file based comparison only) so the printLogsToConsole flag is only there to print the framework diff now (output the contents of .diff file to the console). The configureLogger function configures logger to dump all exception stack-traces (IOException, ParseException etc.) except for SQLException into a .log file, so the flag is unrelated to this function now.